setup.cfg

```diff
@@ -28,4 +28,5 @@ good-names-rgxs = ^[_a-z][_a-z0-9]?$  # allow for 1-character variable names
 
 [pylint.MESSAGE CONTROL]
 disable = all
-enable = line-too-long, invalid-name, pointless-statement
+enable = line-too-long, invalid-name, pointless-statement, missing-function-docstring, unspecified-encoding
+ignore=opensearchpy
```

Comment:
Add space around `=`. Does this file need linting?

Reply:
ha! do we have linting for cfgs? will add the spaces.

Reply:
Not sure!

Reply:
I killed the pylint.Message control config. It's all handled in the lint_per_folder function now